src/api/routes/rosetta/construction.ts

```diff
@@ -165,6 +167,24 @@ export function createRosettaConstructionRouter(db: DataStore, chainId: ChainID)
       }
     }
 
+    // dummy transaction to calculate size
+    const tokenTransferOptions: UnsignedTokenTransferOptions = {
+      recipient: options.token_transfer_recipient_address as string,
+      amount: new BN(options.amount as string),
+      // We don't know the fee yet but need a placeholder
+      fee: new BN(0),
+      // placeholder public key
+      publicKey: '000000000000000000000000000000000000000000000000000000000000000000',
+      network: GetStacksTestnetNetwork(),
```

Comment:
Seems unconventional to uppercase this method. Can it be renamed to `getStacksTestnetNetwork` like the rest?

Reply:
Yes should be ok to do that.

That make me think. Should we not use the correct network info ? We can detect using chain id info right ?

Reply:
I don't think it matters if this tx is only to calculate the size?

Reply:
Yep, the chainID should be used to determine whether testnet or mainnet is used here 👍

Reply:
Yeah this tx is only used to calculate the size but in `/payload` it also calls `GetStacksTestnetNetwork` regardless of the network defined. I am going to create a dedicated PR for this.